app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,39 @@
+class CoursesController < ApplicationController
```

Comment:
Working on it.

Need to merge the course pages first, otherwise I can’t implementmy first dummy module either…